unit-tests/post-processing/test-post-processing-filters.py

```diff
@@ -0,0 +1,469 @@
+    if domain_transform_only:
+        test.check(profile_diffs(diff2orig, 0, 0))
+
+    #Validate the filters
+    #The differences between the reference code and librealsense implementation are byte-compared below
+    return profile_diffs(diff2ref, 0, 0)
+
+def compare_frame_md(origin_depth, result_depth):
+    for i in range(frame_metadata_count):
+        origin_supported = origin_depth.supports_frame_metadata(rs.frame_metadata_value(i))
+        result_supported = result_depth.supports_frame_metadata(rs.frame_metadata_value(i))
+        test.check_equal(origin_supported, result_supported)
+        if origin_supported and result_supported:
+            #FRAME_TIMESTAMP and SENSOR_TIMESTAMP metadatas are not included in post proccesing frames,
+            #TIME_OF_ARRIVAL continues to increase  after post proccesing
+            if i == rs.frame_metadata_value.frame_timestamp or i == rs.frame_metadata_value.sensor_timestamp or i == rs.frame_metadata_value.time_of_arrival:
+                continue
+
+            origin_val = origin_depth.get_frame_metadata(rs.frame_metadata_value(i))
+            result_val = result_depth.get_frame_metadata(rs.frame_metadata_value(i))
+            test.check_equal(origin_val, result_val)
+
+def create_depth_intrinsics(test_cfg):
+    depth_intrinsics = rs.intrinsics()
+    depth_intrinsics.width = test_cfg.input_res_x
+    depth_intrinsics.height = test_cfg.input_res_y
+    depth_intrinsics.ppx = test_cfg.input_res_x / 2.0
+    depth_intrinsics.ppy = test_cfg.input_res_y / 2.0
+    depth_intrinsics.fx = test_cfg.focal_length
+    depth_intrinsics.fy = test_cfg.focal_length
+    depth_intrinsics.model = rs.distortion.brown_conrady
+    depth_intrinsics.coeffs = [0, 0, 0, 0, 0]
+    return depth_intrinsics
+
+def create_video_stream(test_cfg, depth_intrinsics):
+    vs = rs.video_stream()
+    vs.type = rs.stream.depth
+    vs.index = 0
+    vs.uid = 0
+    vs.width = test_cfg.input_res_x
+    vs.height = test_cfg.input_res_y
+    vs.fps = 30
+    vs.bpp = 2
+    vs.fmt = rs.format.z16
+    vs.intrinsics = depth_intrinsics
+    return vs
+
+def create_frame(test_cfg, depth_stream_profile, index):
+    frame = rs.software_video_frame()
+    frame.pixels = test_cfg._input_frames[index]
+    frame.bpp = 2
+    frame.stride = test_cfg.input_res_x * 2
+    frame.timestamp = index + 1
+    frame.domain = rs.timestamp_domain.system_time
+    frame.frame_number = index
+    frame.profile = depth_stream_profile.as_video_stream_profile()
+    frame.depth_units = test_cfg.depth_units
+    return frame
+
+
+################################################################################################
+with test.closure("Post-Processing Filters sequence validation"):
+    test_cfg = ppf_test_config()
+    for first_element, second_element in ppf_test_cases:
+        # Load the data from configuration and raw frame files
+        if not load_test_configuration(first_element, test_cfg):
+            continue
```

Comment:
Maybe we should fail if can't load data?

Reply:
it will fail in the function load_test_configuration, so we can change it but it won't make any change. This is how they did it in the original test